jekyll/_cci2/server/v4.3/installation/phase-2-core-services.adoc

```diff
@@ -679,6 +677,18 @@ object_storage:
     region: "<role-region>"
     irsaRole: "<irsa-arn>"
 ----
+
+**Non-presigned URLs (Optional)**
```

Comment:
```suggestion
**Non-presigned Mode (Optional)**
```
As above, `Non-presigned URLs` doesn't really fit correctly. I think `Non-presigned Mode` is probably the best description, we've also used `Direct Access` internally but I think that might be slightly less clear.